js/script.js

```diff
@@ -0,0 +1,82 @@
+const todoInput = document.querySelector('#todo-input');
+const addTodoBtn = document.querySelector('#add-todo-btn');
+const todoListTitle = document.querySelector('#todo-list-title');
+const doneListTitle = document.querySelector('#done-list-title');
+const todoList = document.querySelector('.todo-list');
+const doneList = document.querySelector('.done-list');
+
+let items = [];
+
+const addNewTodo = (event) => {
+  event.preventDefault();
+  const ret = items.findIndex((item) => item.content === todoInput.value);
+  if (ret === -1 && todoInput.value) {
+    const item = {
+      key: -1,
+      content: todoInput.value,
+      isDone: false,
+    };
+    items.push(item);
```

Comment:
`spread` 연산자를 사용해 보셔도 좋을 것 같습니다!

```suggestion
    items = [...items, item];
```

Reply:
익숙하지 않은 문법이라 직접 코드를 작성할때 사용할 생각을 잘 못하는것 같습니다. 숙달해서 spread 연산자도 사용해보도록 하겠습니다!